src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumInputProgrammer.java

```diff
@@ -17,6 +17,14 @@ public FBASeleniumInputProgrammer(WebDriver driver) {
         this.driver = driver;
     }
 
+    public void seleniumInputProgrammer(String cssSelec, WebElement scopedFBACard, String keys){
```

Comment:
Body looks good :+1:

I would maybe change the name?

```suggestion
    public void programInputFormWithCssSelectorAndKeys(String cssSelec, WebElement scopedFBACard, String keys){
```

Reply:
You can do an `Shift+F6` in Intellij to refactor names automagically across all calls.